skimage/exposure/_adapthist.py

```diff
@@ -75,7 +75,9 @@ def equalize_adapthist(image, kernel_size=None,
         return img_as_float(image)  # convert to float for consistency
 
     image = img_as_uint(image)
-    image = rescale_intensity(image, out_range=(0, NR_OF_GRAY - 1))
+    image = np.round(
+        rescale_intensity(image, out_range=(0, NR_OF_GRAY - 1))
+    ).astype(np.uint16)
```

Comment:
Do we need to worry that the rounding here is a breaking change? See the changed unit test below..

Reply:
@lagru Ah, yes, I'd meant to discuss that. imho rounding is the correct behaviour and I would consider this a bug fix, but we should definitely discuss it! @scikit-image/core any thoughts on this?

Reply:
I agree with @jni that the the rounding is the correct behavior. The best that we can do is may be to add a note in the documentation about this breaking change.

Reply:
> imho rounding is the correct behaviour and I would consider this a bug fix

+1.

Reply:
I've added a note to the release notes and a versionchanged marker to both functions.